### DISCUSSION TITLE
How often does plugin sync with provider?

### DISCUSSION THREAD

**Discussion by fazar7** [vosmiic/jellyfin-ani-sync#120] [Q&A] [answered]
Hey there, just got the plugin setup and running and I noticed it mass added everything from my server to my provider (anilist) 
but when I watch something, it doesnt (immediately) update, im not sure if its set on a timer (every x hours maybe) or maybe I just messed something up. 
Any help is appreciated.

 
Running jf Version: 10.8.10 on windows

**Comment by vosmiic** [comment, marked as answer]
Thanks for creating a disucssion for this.
It should sync your progress with the provider quite quickly (the longest that I have seen it take was probably around 10 seconds) after watching an episode/movie. That being said, after watching an episode/movie is there anything notable logged by the plugin in the logs?

**Reply by fazar7** [reply]
Hey there, so after some more testing, I had to enable the "Ticking watched updates providers?" to get it to work, without that enabled, it doesnt show up on my provider. 
Ive also not had any crashes so I think this has fixed my issue. (im pretty sure I was just dense before lol).
So again, thank you so much haha, the plugins come in clutch honestly.

**Reply by vosmiic** [reply]
All good, glad that you got it working and the plugin is working for you, thanks!
